site/content/en/docs/Reference/API/images.md

```diff
@@ -0,0 +1,169 @@
+| `newName` | Override the image **name** for images whose image name matches `name`   | `newName: nginx-special` | `nginx:old` -> `nginx-special:old` |
+
+
+## Example
+
+### File Input
+
+```yaml
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: the-deployment
+spec:
+  template:
+    spec:
+      containers:
+      - name: mypostgresdb
+        image: postgres:8
+      - name: nginxapp
+        image: nginx:1.7.9
+      - name: myapp
+        image: my-demo-app:latest
+      - name: alpine-app
+        image: alpine:3.7
+
+```
+
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+- name: postgres
+  newName: my-registry/my-postgres
+  newTag: v1
+- name: nginx
+  newTag: 1.8.0
+- name: my-demo-app
+  newName: my-app
+- name: alpine
+  digest: sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3
+
+resources:
+- deployment.yaml
+
+```
+
+### Build Output
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: the-deployment
+spec:
+  template:
+    spec:
+      containers:
+      - image: my-registry/my-postgres:v1
+        name: mypostgresdb
+      - image: nginx:1.8.0
+        name: nginxapp
+      - image: my-app:latest
+        name: myapp
+      - image: alpine@sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3
+        name: alpine-app
+```
+
+## Setting a Name
+
+The name for an image may be set by specifying `newName` and the name of the old container image.
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - name: mycontainerregistry/myimage
+    newName: differentregistry/myimage
+```
+
+## Setting a Tag
+
+The tag for an image may be set by specifying `newTag` and the name of the container image.
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - name: mycontainerregistry/myimage
+    newTag: v1
+```
+
+## Setting a Digest
+
+The digest for an image may be set by specifying `digest` and the name of the container image.
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - name: alpine
+    digest: sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3
+```
+
+
+## Setting a Tag from the latest commit SHA
+
+A common CI/CD pattern is to tag container images with the git commit SHA of source code.  e.g. if
+the image name is `foo` and an image was built for the source code at commit `1bb359ccce344ca5d263cd257958ea035c978fd3`
+then the container image would be `foo:1bb359ccce344ca5d263cd257958ea035c978fd3`.
+
+A simple way to push an image that was just built without manually updating the image tags is to
+download the [kustomize standalone](https://github.com/kubernetes-sigs/kustomize/) tool and run
```

Comment:
Suggest linking to this relative path: `/docs/getting-started/installation/`

Reply:
Fixed it.
https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/a2b60e4bcb498a8e69b48fdb24561b9d09b5df18#diff-f47a78106551e0be110689bfedf57eab270acb71664405e1892c336a3d1b1bbbR143